services/garage-backend/src/helpers/multerUpload.ts

```diff
@@ -0,0 +1,36 @@
+import multer from 'multer';
+import * as fs from 'fs';
+import { randomUUID } from 'crypto';
+
+const storage = multer.diskStorage({
+  destination(req, file, cb) {
+    const dir = './uploads/';
+    fs.exists(dir, (exist) => {
+      if (!exist) {
+        return fs.mkdir(dir, (error) => cb(error, dir));
+      }
+      return cb(null, dir);
+    });
+  },
+
+  filename(req: any, file: any, cb: any) {
```

Comment:
```suggestion
  filename(req, file, cb) {
```
`any` should not be ever used, only in emergencies when something does not behave as expected